client/src/components/categorical/category/index.tsx

```diff
@@ -450,6 +450,7 @@ const CategoryHeader = React.memo(
             alignItems: "flex-start",
           }}
         >
+          {/* eslint-disable-next-line jsx-a11y/label-has-associated-control -- TODO: Need to separate expand and checkbox label */}
```

Comment:
Add `eslint-disable-next-line jsx-a11y/label-has-associated-control` for the whole category label, because it's violating a11y due to using one label for both checkbox and expand/collapse the whole category. So we need UX involved for a complete solution 👌

Reply:
All of my attempts to treat this error were affecting UI - that's why I opted just to mute it, for now.

Thanks for tackling this and further investigating